compose/ui/ui/build.gradle

```diff
@@ -170,6 +170,8 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
                 api project(":compose:ui:ui-unit")
                 api project(":compose:ui:ui-util")
                 implementation(project(":annotation:annotation"))
+                implementation(project(":lifecycle:lifecycle-common"))
```

Comment:
Because we add a new dependency, this change requires 2 approves, please ask someone else to review this commit (about adding dependency).

It looks fine for me, because we either will release a stable version of `lifecycle`, or we just depend on stable API of it right now.

Reply:
@eymar, no need to review the whole PR, only this commit

Reply:
> there was an idea

If you talk about my comment:

>If we don't need it urgently

I meant that "if don't need it right now". But we definitely need it for the 1.6.10 release

Reply:
even dev builds won't work untill lifecycle is published if this PR is merged before that.

Reply:
Indeed. Then let's wait until we publish